src/src/PluginStructs/P093_data_struct.cpp

```diff
@@ -362,6 +382,26 @@ void P093_data_struct::applySettings() {
   packet[21] = checkSum(packet, 21);
 
   sendPacket(packet, PACKET_LEN);
+
```

Comment:
I need Help here...

When i put line 362 to 364 under line 404 remote temperature works, but not the other commands.. if these lines are like this the other commands work, but not remote temperature...

What am I missing???

Sorry I am not a programmer, i try just to help myself out on an issue i opened long time ago...

Thx for your help...

Reply:
I assume you are _moving_ those lines from 382-384 to below where the `remoteTemperature` value is put into the packet buffer?
This could mean that, besides the other commands, _also_ the remoteTemperature is put into the packet buffer, causing the other commands to be overwritten/changed by that data.
Some generic improvements can be made to this plugin, like initializing float values with a float 0 constant: `0.0f`, and compare these values using a special float comparator, I'll mark those lines in the .h file, below (and can also be applied to the other 2 float values).

Reply:
Yes that´s right.. I also tried with another buffer called packet2, which is just handled after the if (_writeStatus.isDirty(RemoteTemperature)) section.. but same behaviour, i can set the hp on /off etc, but not the remote temp. I can just set remote temperature when it is buffer called packet behind if (_writeStatus.isDirty(RemoteTemperature))... 
I also tried to set the buffer in each if(_writeStatus.isDirty(...)) and just execute sendPacket(packet, PACKET_LEN); in the end of this block.. but than only remote remote is handled no other command...